sympy/printing/tests/test_repr.py

```diff
@@ -101,12 +101,11 @@ def test_Rational():
 
 
 def test_Float():
-    sT(Float('1.23', prec=3), "Float('1.22998', prec=3)")
```

Comment:
It can be explained because an extra padding of 3 decimal digits is provided to make sure that no precision in lost between translations from decimal to binary(at least I think that's the case). Since now the translation from decimal to binary precision is taken care of by the `dps_to_prec` call in the `Float` constructor, there is no need for it.
```
def repr_dps(n):
    """Return the number of decimal digits required to represent
    a number with n-bit precision so that it can be uniquely
    reconstructed from the representation."""
    dps = prec_to_dps(n)
    if dps == 15:
        return 17
    return dps + 3
```